python/cudf/cudf/pandas/scripts/run-pandas-tests.sh

```diff
@@ -16,12 +16,13 @@
 #
 # This script creates a `pandas-testing` directory if it doesn't exist
 
+set -euo pipefail
 
 # Grab the Pandas source corresponding to the version
 # of Pandas installed.
 PANDAS_VERSION=$(python -c "import pandas; print(pandas.__version__)")
 
-PYTEST_IGNORES="--ignore=tests/io/test_user_agent.py"
+PYTEST_IGNORES="--ignore=tests/io/test_user_agent.py --ignore=tests/interchange/test_impl.py"
```

Comment:
Can we just inline this variable? It's only used once, it's defined far from that location (the pytest call), and it's incongruous with how we provide basically every other argument inline.

Reply:
It's "far" but not really that far. It's just that there's another very large variable definition in between. I think I'm going to leave this as-is because I don't think it is worth another CI run to change.